### PR TITLE
Quadrat: Add Latest episodes block pattern

### PR DESCRIPTION
**Pull request by MaggieCabrera**
#### Changes proposed in this Pull Request:

This PR adds the "Latest Episodes" block pattern. This pattern uses one new utility class to remove the bottom margin from the image block so that it sits flush with the following block. To do: upload the images to the staging website. ✅ 

Screenshot 2021-04-16 at 12 39 40

#### Related issue(s):

Partially addresses #3642

### CODE REVIEW

File: quadrat/inc/block-patterns.php [hunk 1]
```diff
@@ -27,6 +27,47 @@ function quadrat_register_block_patterns() {
...
 
+			register_block_pattern(
+				'quadrat-blocks/example',
+				array(
+					'title'      => __( 'Latest Episodes', 'quadrat' ),
+					'categories' => array( 'quadrat' ),
+					'content'    => '<!-- wp:columns {"align":"wide"} -->
+					<div class="wp-block-columns alignwide"><!-- wp:column -->
+					<div class="wp-block-column"><!-- wp:image {"id":1438,"sizeSlug":"large","linkDestination":"none","className":"image-no-margin mb-0"} -->
+					<figure class="wp-block-image size-large image-no-margin mb-0"><img src="http://freethemes.local/wp-content/uploads/2021/04/Screen-usage-1024x623.jpg" alt="' . esc_html__( 'Screen usage', 'quadrat') . '" class="wp-image-1438"/></figure>
```
Comment (MaggieCabrera) [inline quadrat/inc/block-patterns.php]: We probably need a better alt text than this

Comment (MaggieCabrera) [reply quadrat/inc/block-patterns.php]: Also, @beafialho should we keep the lorem ipsum or do we want other kind of text for these?

Comment (beafialho) [reply quadrat/inc/block-patterns.php]: Yes, @MaggieCabrera here's the alt text for the images of the patterns. I'm also writing alt text for all the other abstract illustrations [here](https://www.figma.com/file/EDoXTvsZt4lMHItzOczFQw/Quadrat?node-id=111%3A34).

"Illustration of two women."
Illustration of two women

"Illustration of an introspective woman."
Illustration of an introspective woman

"Illustration of a woman working on a laptop."
Illustration of a woman working on a laptop

"Illustration of a woman climbing steps."
Illustration of a woman climbing steps

> Also, @beafialho should we keep the lorem ipsum or do we want other kind of text for these?

I've replaced the lorem ipsum [here](https://www.figma.com/file/EDoXTvsZt4lMHItzOczFQw/Quadrat?node-id=443%3A5).

Comment (MaggieCabrera) [reply quadrat/inc/block-patterns.php]: Thanks so much!

**Comment by jffng** [comment]
On mobile, should these have some vertical margin?

Screen Shot 2021-04-16 at 12 26 59 PM

If so, I wonder if that's something more global with the columns block CSS, or maybe we can wrap them in a group, I'm not sure.

File: quadrat/sass/_utility.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.image-no-margin {
```
Comment (jffng) [inline quadrat/sass/_utility.scss:1]: Nice candidate to move to BCB eventually?

**Comment by beafialho** [comment]
> On mobile, should these have some vertical margin?
> 
> Screen Shot 2021-04-16 at 12 26 59 PM
> 
> If so, I wonder if that's something more global with the columns block CSS, or maybe we can wrap them in a group, I'm not sure.

I think this is very similar to what I had mocked up, but would like @kjellr's input here.

**Comment by kjellr** [comment]
> If so, I wonder if that's something more global with the columns block CSS, or maybe we can wrap them in a group, I'm not sure.

I do think it makes sense to add a little top/bottom margin. Do they have margin in the editor? We should also ensure it matches in both places.

File: quadrat/sass/_utility.scss (hunk 2 above)
Comment (MaggieCabrera) [reply quadrat/sass/_utility.scss:1]: Yes, I think so, I found out there's a .com plugin that does exactly this but adding a toggle inside the editor!

**Comment by MaggieCabrera** [comment]
I added a group block wrapper with 30px padding bottom. 

The editor (the front is the same):

Screenshot 2021-04-22 at 15 27 45

The frontend on mobile:

Screenshot 2021-04-22 at 15 27 52

**Review by kjellr — COMMENTED** [review]
This looks great! Just one small comment: the body text in here should have a line height of `1.7`. Once that's all set, this should be good to go.

**Comment by jffng** [comment]
Nice work, thanks everyone!

**Pull request merged**